paasta_tools/cli/fsm/questions.py

```diff
@@ -121,39 +120,14 @@ def get_deploy_stanza():
         {"instancename": "security-check", },
         {"instancename": "push-to-registry", },
         {"instancename": "performance-check", },
-        {"instancename": "pnw-stagea.canary", },
-        {"instancename": "pnw-stagea.main", },
-        {"instancename": "norcal-stageb.canary", },
-        {"instancename": "norcal-stageb.main", },
-        {"instancename": "norcal-devb.canary", },
-        {"instancename": "norcal-devb.main", },
-        {"instancename": "norcal-devc.canary", },
-        {"instancename": "norcal-devc.main", "trigger_next_step_manually": True, },
-        {"instancename": "norcal-prod.canary", },
-        {"instancename": "nova-prod.canary", "trigger_next_step_manually": True, },
-        {"instancename": "norcal-prod.main", },
-        {"instancename": "nova-prod.main", },
+        {"instancename": "dev.everything", },
```

Comment:
While you're touching this stuff, can you pull the data into a system config so that fsm can be more useful to non-Yelp people?